src/applications/caregivers/tests/unit/components/FormFields/FacilityList.unit.spec.jsx

```diff
@@ -8,15 +8,16 @@ import FacilityList from '../../../../components/FormFields/FacilityList';
 
 describe('CG <FacilityList>', () => {
   const subject = ({
-    reviewMode = false,
-    submitted = false,
+    reviewMode,
+    submitted,
+    context = undefined,
     value = undefined,
     error = undefined,
     onChangeSpy = () => {},
   } = {}) => {
     const props = {
       facilities: mockFetchFacilitiesResponse.facilities,
-      formContext: { reviewMode, submitted },
+      formContext: context === undefined ? { reviewMode, submitted } : context,
```

Comment:
Just tricking the setup here to account for when the `formContext` value could be falsy.